src/StockportWebapp/Views/stockportgov/Shared/GlobalAlerts.cshtml

```diff
@@ -12,4 +12,24 @@
             <partial name="GlobalAlert" model='alert' />
         }
     }
-}
\ No newline at end of file
+}
+
+<script type="text/javascript" data-cookie-consent="strictly-necessary">
+    require(['/assets/javascript/config-80795c2e.min.js'], function () {
+        require(['alerts'],
+            function (alerts) {
+                alerts.Init();
+            }
+        );
+    });
+</script>
+
+<script type="text/plain" data-cookie-consent="functionality">
+    require(['/assets/javascript/config-80795c2e.min.js'], function () {
+        require(['alertsSetCookie'],
+            function (alertsSetCookie) {
+                alertsSetCookie.Init();
+            }
+        );
+    });
+</script>
```

Comment:
Could you call the AlertInit partial instead?

Reply:
Done